.backportrc.json

```diff
@@ -2,6 +2,7 @@
   "targetBranchChoices": [
```

Comment:
These changes need to go to `main`

Reply:
@artem-shelkovnikov I have zero familiarity with this project so I need better directions. Does the entire PR need to be merged against main, or do I need to split this PR into two with some changes going to 8.x and some going to main? And in that case, which changes go to each branch?

Reply:
I've added this change for main on a separate PR https://github.com/elastic/connectors/pull/2893

Reply:
Sorry just seen your response - CI-related things normally stay on `main` - so `catalog-info.yaml` changes just go there.

Not sure about this file - `.backportrc.json` TBH cause we might need to do backports `8.x` -> `8.something`. So merging this file in `8.x` would not hurt, but it might not affect anything